src/pages/index.js

```diff
@@ -1,129 +1,218 @@
+  const avatarValidator = new FormValidator(Constants.configuration, Constants.avatarForm);
+  const user = new UserInfo(Constants.profileName, Constants.profileDescription, Constants.avatarPhoto);
+
 
   editValidator.enableValidation();
   addValidator.enableValidation();
-
-  const profile = document.querySelector("#profile");
-  const profileEditButton = profile.querySelector("#profile__edit_button");
-  const profileAddButton = profile.querySelector("#profile__add_button");
-  const profileName = profile.querySelector("#profile__name");
-  const profileDescription = profile.querySelector("#profile__description");
+  avatarValidator.enableValidation();
 
   const imagePopup = new PopupWithImage("#picture-modal");
 
   const editPopup = new PopupWithForm(
-    `#${editForm.parentElement.id}`,
+    `#${Constants.editForm.parentElement.id}`,
     handleProfileEditSubmit
   );
   const addPopup = new PopupWithForm(
-    `#${addForm.parentElement.id}`,
+    `#${Constants.addForm.parentElement.id}`,
     handleProfileAddSubmit
   );
-  const userInfo = new UserInfo(profileName, profileDescription);
 
   imagePopup.setEventListeners();
 
+  api
+    .getProfileInfo()
+    .then((results) => {
+      user.setUserInfo({ name: results.name, about: results.about });
+      user.setAvatar(results.avatar);
+    })
+    .catch((error) => {
+      console.error("Caught an error: ", error.message);
+    });
+
   //functions
+  const deletePopup = new confirmationPopup("#confirmation-modal");
+  deletePopup.setEventListeners();
+
+  function addCard(name, description, template, id) {
+    const items = [{name: name, description:description}];
+    const cardsSection = new Section(
+      {
+        items,
+        renderer: (cardData) => {
+          addCard(cardData.name, cardData.link, Constants.cardTemplate, cardData._id);
+        },
+      },
+      Constants.galleryList
+    );
+    const modelCard = new Card(
+      name,
+      description,
+      template,
+      imageClickHandler,
+      (card) => {
+
+        deletePopup.open();
+        deletePopup.setDefaultSaveButton();
+        deletePopup.setSubmitAction(() => {
+          api
+            .deleteCard(card._id)
+            .then(() => {
+              card.deleteCard();
+            })
+            .then(() =>{
+              deletePopup.updateSaveButton()
+            })
+            .then(() =>{
+              deletePopup.close()
+            })
+            .catch((error) => {
+              console.error("Caught an error: ", error.message);
+            });
+        });
+      },
+      toggleLikeButton,
+      initialCardsInfo,
+      id
+    );
+
+    cardsSection.addItem(modelCard.generateCard())
+  }
 
   function createCard(name, description, template) {
-    const modelCard = new Card(name, description, template, imageClickHandler);
-    cardsSection.addItem(modelCard.addCard());
+    addPopup._submitBtn.textContent = "Saving...";
```

Comment:
`_submitBtn`

Please, delete the **low dash** (`underscore`) from methods if you want to use them `outside` a class because the `low dash` shows a **private** method which is used only inside the class.


**Please, correct it everywhere**